server/balancer.go

```diff
@@ -22,8 +23,30 @@ import (
 
 var storeCacheInterval = 30 * time.Second
 
+// minBalanceDiff returns the minimal diff to do balance. The formula is based
+// on experience to let the diff increase alone with the count slowly.
+func minBalanceDiff(count uint64) float64 {
+	return math.Pow(2, math.Log10(float64(count)))
```

Comment:
It will limit diff to 32 for 100k regions, seems too small.
How about `sqrt(count)`?